hack/rhcos-sas/rhcos-sas.go

```diff
@@ -27,12 +27,12 @@ func run(ctx context.Context, log *logrus.Entry) error {
 	resourceGroup := "images"
 	accountName := "openshiftimages"
 
-	authorizer, err := auth.NewAuthorizerFromEnvironment()
+	env, err := env.NewCore(ctx, log)
```

Comment:
```suggestion
_env, err := env.NewCore(ctx, log)
```

if you use a variable name same as the package you're importing, it can lead to some interesting things later on.  

We currently do it in a few places, but should fix it up.

Reply:
Will need to update the usage of the variable below as well.

Reply:
will do, great catch!